spark/src/main/kotlin/com/adevinta/spark/components/progress/CircularProgressIndicator.kt

```diff
@@ -42,8 +42,7 @@ import androidx.compose.material3.CircularProgressIndicator as MaterialCircularP
 @InternalSparkApi
 @Composable
 internal fun SparkCircularProgressIndicator(
-    @FloatRange(from = 0.0, to = 1.0)
-    progress: Float,
+    progress: () -> Float,
```

Comment:
Q: does the annotation breaks the new type?

Reply:
Nope but they can't verify that the value passed is in the range now.
The progress value was already coerced in this range but now we won't have a warning if it's not inside it